src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -375,6 +379,87 @@ public DataAccessRequest cancelDataAccessRequest(String referenceId) {
                 user);
     }
 
+    /**
+     * Generate a list of DARs split by dataset id. Generate one DAR per dataset that is
+     * being requested. In the case of a single dataset DAR, update the existing value.
+     * In the case of multiple dataset DARs, update the first one and create new ones for each
+     * additional dataset past the first.
+     *
+     * @param user The User
+     * @param dataAccessRequest DataAccessRequest with populated DAR data
+     * @return List of created DARs.
+     */
+    public List<DataAccessRequest> createDataAccessRequest(User user, DataAccessRequest dataAccessRequest) {
+        Date now = new Date();
+        List<DataAccessRequest> newDARList = new ArrayList<>();
+        DataAccessRequestData darData = dataAccessRequest.getData();
+        darData.setPartialDarCode(null);
+        darData.setUserId(user.getDacUserId());
+        if (Objects.isNull(darData.getCreateDate())) {
```

Comment:
Minor nitpick, it might be a better idea to save now.getTime() to a variable and then assign that as an argument to setCreateDate and setSortDate as opposed to passing in two separate calls of getTime()

Reply:
Also another question: what does sortDate represent? The naming isn't really clear to me, from what I've read from the code it acts like an update timestamp, but I'm not sure

Reply:
I'm adding some comments around the sort date. For a preview:
```
  /**
   * Legacy property on DARs. Used to display the sort order for a DAR.
   * In practice, this also functions as the Update Date.
   * See also https://broadinstitute.atlassian.net/browse/DUOS-728
   */
  @JsonProperty public Date sortDate;
```

Reply:
Done.